### ISSUE TITLE
Ability to add a new site without re provisioning the box

### ISSUE THREAD

**Issue by cycomarcooo** [ezekg/theme-juice-cli#61]
It should be useful to add a site without re-provisioning the entire VM. For now, several packages and scripts are updated during provision which is not necessary wanted. The idea is to "lock" the server stack at a known state whenever the user add a new site.

**Comment by ezekg** [comment]
I'm not a fan of having an outdated development box, however I understand that you might not want to trigger updates on every provision—me either—so I think that figuring out a way to split these up might be a good idea. Maybe we can set up most of the non-update work for the box to be run on a `reload` instead, i.e. the things required for setting up a new site.

In any case, this should be an issue on the VM repo, since it doesn't concern the CLI. :+1:

**Comment by cycomarcooo** [comment]
You're definitely right. I dug a bit the subject and finally understood I need to create a Vagrant package that is close to the managed VPS I'm deploying to, with respect of tj [structure needs](https://github.com/ezekg/theme-juice-cli#so-does-that-mean-i-can-use-any-vagrant-box). I created a very small Vagrant file with obviously all the sync stuff between the host and guest (db, vhost, root folders) and only one install/update: wp-cli inside /srv/www.
Then I use tj with the --vm-path option to specify the box I'd like to use. It works like a charm.

BTW, with this solution, I realised the [connection problem I mentioned in a previous issue](https://github.com/ezekg/theme-juice-vvv/issues/2#issuecomment-206790747) is gone.
